database-plugins/src/main/java/io/cdap/plugin/DBManager.java

```diff
@@ -53,17 +53,20 @@ public DBManager(ConnectionConfig config) {
       config.connectionArguments), config.jdbcPluginType);
   }
 
-  @Nullable
-  public Class<? extends Driver> validateJDBCPluginPipeline(PipelineConfigurer pipelineConfigurer,
-                                                            String jdbcPluginId, FailureCollector collector) {
+  public void validateCredentials(FailureCollector collector) {
     if (!config.containsMacro(DBConfig.USER) && !config.containsMacro(DBConfig.PASSWORD)
-      && config.getUser() == null && config.getPassword() != null) {
+          && config.getUser() == null && config.getPassword() != null) {
```

Comment:
nit: space?

Reply:
The original indentation was off, since this is separated for a if condition, the indentation should based on the start of the condition